android/app/src/main/java/org/fossasia/openevent/data/Speaker.java

```diff
@@ -16,167 +19,315 @@
+    @Expose
+    private String mobile;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("organisation")
+    @Expose
+    private String organisation;
+    @SerializedName("photo")
+    @Expose
+    private String photo;
+    @SerializedName("position")
+    @Expose
+    private String position;
     @SerializedName("sessions")
-    int[] session;
-
-    String country;
+    @Expose
+    private List<Session> sessions = new ArrayList<Session>();
+    @SerializedName("short_biography")
+    @Expose
+    private String short_biography;
+    @SerializedName("twitter")
+    @Expose
+    private String twitter;
+    @SerializedName("website")
+    @Expose
+    private String website;
+
+    /**
+     * No args constructor for use in serialization
+     */
+    public Speaker() {
+    }
 
-    public Speaker(int id, String name, String photo,
-                   String bio, String email, String web,
-                   String twitter, String facebook, String github,
-                   String linkedin, String organisation,
-                   String position, int[] session, String country) {
-        this.id = id;
-        this.name = name;
-        this.photo = photo;
-        this.bio = bio;
+    /**
+     * @param country
+     * @param email
+     * @param facebook
+     * @param github
+     * @param id
+     * @param linkedin
+     * @param long_biography
+     * @param name
+     * @param organisation
+     * @param photo
+     * @param position
+     * @param twitter
+     * @param website
+     */
+    public Speaker(Integer id, String name, String photo, String long_biography, String email, String website, String twitter, String facebook, String github, String linkedin, String organisation,  String position, List<Session> sessions, String country) {
+        this.country = country;
         this.email = email;
-        this.web = web;
-        this.twitter = twitter;
         this.facebook = facebook;
         this.github = github;
+        this.id = id;
         this.linkedin = linkedin;
-        this.organisation = organisation;
-        this.position = position;
-        this.session = session;
-        this.country = country;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
+        this.long_biography = long_biography;
         this.name = name;
-    }
-
-    public int[] getSession() {
-        return session;
-    }
-
-    public void setSession(int[] session) {
-        this.session = session;
-    }
-
-    public String getPhoto() {
-        return photo;
-    }
-
-    public void setPhoto(String photo) {
+        this.organisation = organisation;
         this.photo = photo;
+        this.position = position;
+        this.sessions = sessions;
+        this.twitter = twitter;
+        this.website = website;
     }
 
-    public String getBio() {
-        return bio;
+    /**
+     * @return The country
```

Comment:
No need for these comments for params, if this is inserted by android studio, you can change the settings and avoid such comments in the future